test/e2e/scenarios/Cluster/cluster_scale_in_scale_up_increase_replica.yaml

```diff
@@ -46,6 +56,14 @@ steps:
           ACK.ResourceSynced:
             status: "True"
             timeout: 10000
+    expect_k8s:
+      spec:
+        nodeType: db.t4g.medium
+        numShards: 1
+        numReplicasPerShard: 1
+    expect_aws:
+      NodeType: db.t4g.medium
+      NumberOfShards: 1
```

Comment:
nit: assert num replicas too

Reply:
I don't find a field related to number of replicas from `describe_clusters` API called by boto3.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/memorydb/client/describe_clusters.html